doc/conf.py

```diff
@@ -748,6 +748,8 @@ def append_attr_meth_examples(app, what, name, obj, options, lines):
     # Too slow
     "https://speakerdeck.com/dengemann/",
     "https://www.dtu.dk/english/service/phonebook/person",
+    # Can't find a replacement
+    "https://github.com/rubyyhj",
```

Comment:
Based on https://github.com/mne-tools/mne-python/pull/8580, I think they changed GH username to https://github.com/hongjiang-ye